src/main/java/org/retest/rebazer/connector/BitbucketConnector.java

```diff
@@ -40,11 +42,12 @@ public BitbucketConnector( final RepositoryConfig repoConfig, final RestTemplate
 	@Override
 	public PullRequest getLatestUpdate( final PullRequest pullRequest ) {
 		final DocumentContext jsonPath = jsonPathForPath( requestPath( pullRequest ) );
+		final Date repositoryTime = Date.from( OffsetDateTime.parse( jsonPath.read( "$.updated_on" ) ).toInstant() );
```

Comment:
extract `Date.from( OffsetDateTime.parse( string ).toInstant() );` to method, it exists twice.